package.json

```diff
@@ -51,13 +51,17 @@
     "@types/mocha": "^8.0.3",
     "@types/node": "^14.14.2",
     "@types/puppeteer": "~3.0.2",
+    "@typescript-eslint/eslint-plugin": "^4.5.0",
+    "@typescript-eslint/parser": "^4.5.0",
     "basic-auth": "^2.0.1",
     "eslint": "^7.12.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsdoc": "^30.7.3",
```

Comment:
this plugin seems to be a default plugin added by tslint-to-eslint.

We are not using jsdoc anywhere on wendigo (and don't plan to for the moment) Could you remove this dependency along with the related rules in the eslint-file, please?

Reply:
ah yeh, good point. i've removed 👍